lib/src/phy/dft/dft_fftw.c

```diff
@@ -33,6 +37,13 @@
 #define dft_ceil(a, b) ((a - 1) / b + 1)
 #define dft_floor(a, b) (a / b)
 
+#ifdef USE_CUFFTW
+
+#define FFTW_R2HC 0
```

Comment:
could you leave a comment here and explain the meaning for both?

Reply:
* Right, not needed so removed.
Had the wrong impression cuFFTW exports under different names.